src/Microsoft.AspNetCore.Razor.Language/RazorProjectEngine.cs

```diff
@@ -20,60 +20,45 @@ public abstract class RazorProjectEngine
 
         public static RazorProjectEngine Create(RazorProjectFileSystem fileSystem) => Create(fileSystem, configure: null);
 
-        public static RazorProjectEngine Create(RazorProjectFileSystem fileSystem, Action<RazorProjectEngineBuilder> configure)
+        public static RazorProjectEngine Create(RazorProjectFileSystem fileSystem, Action<RazorProjectEngineBuilder> configure) => Create(fileSystem, configure, RazorConfiguration.Default);
```

Comment:
What would you think about just providing the third-arg version of this?

Reply:
ehhh, not a fan. You'll rarely use the third overload outside of Razor so I'd prefer not to force any extensibility pieces to use that.

Reply:
> force

You keep using this word 😆  It's not a burden to be explicit about something if it's like 10 characters (the configuration).  I also super don't understand who would use this without providing **any** setup for the builder.

I don't want to block your PR on this issues, let's resolve to talk about it next week. I think we're repeating some mistakes made in the first version of this code.